client/src/components/CommentApp/components/CommentReply/index.tsx

```diff
@@ -105,17 +105,17 @@ export default class CommentReplyComponent extends React.Component<CommentReplyP
             value={reply.newText}
             onChange={onChangeText}
           />
-          <div className="comment-reply__actions">
+          <div className="comment__actions">
```

Comment:
See stylesheets – reusing the same class names as for comments, as the styles are almost identical.